<!--
    Launcher for basic stereo camera which publishes images
-->
<launch>
  <!-- Machine settings. See http://www.ros.org/wiki/roslaunch/XML/machine -->
  <!--machine name="local_alt" address="localhost" default="true" /-->
  <!-- stereocamera -->
  <node pkg="minorucam" type="minorucam_laser" name="minorucam_laser" respawn="true" output="screen">
    <!-- left camera device -->
    <param name="dev_left" value="/dev/video1" />
    <!-- right camera device -->
    <param name="dev_right" value="/dev/video0" />
    <!-- image width -->
    <param name="width" value="320" />
    <!-- image height -->
    <param name="height" value="240" />
    <!-- Frames per second -->
    <param name="fps" value="30" />
    <!-- Flip the left image -->
    <param name="flip_left" value="false" />
    <!-- Flip the right image -->
    <param name="flip_right" value="false" />
    <!-- Enable histogram equalization -->
    <param name="hist_equal" value="false" />
    <!-- Enable publishing raw left image -->
    <param name="pub_image" value="true" />
    <!-- Laser scan parameters -->
    <param name="scan_angle_min" value="-1.57" />
    <param name="scan_angle_max" value="1.57" />
    <param name="scan_angle_increment" value="0.00314" />
    <param name="scan_range_min" value="500" />
    <param name="scan_range_max" value="10000" />
    <param name="scan_height_min" value="-10" />
    <param name="scan_height_max" value="10" />
    <!-- Calibration -->
    <param name="calibration_filename" value="/home/pi/calibration.txt" />

    <!-- Enable Profiling -->
    <param name="profiling" value="false" />

  </node>
</launch>

